packages/parser/src/language/sankey/sankey.langium

```diff
@@ -20,4 +20,7 @@ SankeyLink:
 ;
 
```

Comment:
(minor)
The `SANKEY_LINK_VALUE` lines should go _after_ the `SANKEY_LINK_NODE` line to be consistent with the order in the `SankeyLink:` line.

Reply:
And regarding the order of the `SANKEY_LINK_NODE` and `SANKEY_LINK_VALUE` rules, I was trying to make the point that since the `SankeyLink` rule 
```
SankeyLink:
  source=SANKEY_LINK_NODE "," target=SANKEY_LINK_NODE ","  value=SANKEY_LINK_VALUE EOL
;
```
has `SANKEY_LINK_NODE` _first_ in it and `SANKEY_LINK_VALUE` _after_  `SANKEY_LINK_NODE` , the specific rules for each of those (`..._NODE` and `...VALUE`) should _also_ appear in that order in the grammar file.  It is keeping things ordered consistently: when you see things that are in a certain order, they should appear _in that same order_ elsewhere (unless there's a reason they can't, e.g. some technical reason).  

This grammar file is so trivial that it's not a big deal. At all.  (which is why I called it "minor")

Reply:
Sorry for the late reply. You're right, the `SANKEY_LINK_NODE` rule would to occur more than `SANKEY_LINK_VALUE`, so it would make it a bit faster to parse and tokenize the document.

Reply:
Done 3597ffe54a17680e830b154d39ea5495e6dd78d6